CHANGELOG.md

```diff
@@ -28,6 +28,23 @@
 
   ([Giacomo Cavalieri](https://github.com/giacomocavalieri))
 
+- The compiler will now raise a warning when using deprecated syntax to
+  prepend to a list:
```

Comment:
```suggestion
- The syntax `[a..b]` is now deprecated in favour of the `[a, ..b]` syntax.
  This was to avoid it being mistaken for a range syntax, which Gleam does
  not have.
```